src/TelemetryChannels/ServerTelemetryChannel/Shared/AdaptiveSamplingTelemetryProcessor.cs

```diff
@@ -71,6 +71,16 @@ public string ExcludedTypes
         }
 
         /// <summary>
+        /// Gets or sets a semicolon separated list of telemetry types that should be sampled. All types are sampled when left empty.
+        /// </summary>
+        public string IncludedTypes
+        {
+            get { return this.samplingProcessor.IncludedTypes; }
```

Comment:
should setting of IncludeTypes reset ExcludeTypes and vice versa? The behavior is confusing when both are set